### DISCUSSION TITLE
Character Lora doesn't work well in Fooocus while it works good on Automatic1111. What's causing the diff?

### DISCUSSION THREAD

**Discussion by zackzhuang1996** [lllyasviel/Fooocus#1283] [Q&A]
So I trained a character Lora with khoya SS script and I use it in fooocus. It was working well with shorter prompt but when I have a longer prompt, it doesn't really resemble the character anymore. I passed the same prompt to Webui Automatic1111 and it still works well. I wonder if there is any diff in the prompt processing step that causes such a difference and if yes, is there any way I can fix this? I didn't use Fooocus V2 or other styles in the process, except Fooocus negative. 

Here are the prompts and parameters I used for my image generation. zxc person is my lora trigger word. 

`prompt: (close up shot:1.5) Medium Shot, zxc person, poised and determined, glasses glinting, lab coat billowing as she strides purposefully, ponytail swaying, the confident step of someone on a mission 1950s American City street, zxc person Often wears a white lab coat over a knee-length floral dress, matching gloves, cat-eye glasses, and sensible brown oxfords. Her blonde hair is curled and kept neatly in a ponytail.. emotional, harmonious, vignette, highly detailed, high budget, bokeh, cinemascope, moody, epic, gorgeous, film grain, grainy, negative_prompt: , sharpness: 2, performance_selection: Speed, aspect_ratios_selection: 768 1344, image_number: 1, image_seed: 879657518, guidance_scale: 4, base_model_name: dreamshaperXL10_alpha2Xl10.safetensors, refiner_model_name: None, refiner_switch: 0.5, loras: [{'model_name': 'add-detail-xl.safetensors', 'weight': 0.5}, {'model_name': 'character_lora.safetensors', 'weight': 0.75}], require_base64: True, async_process: False, style_selections: ['Fooocus Negative']`

**Comment by FrietbakkerThjeu** [comment]
Did you train it in SDXL or SD1.5?

**Reply by zackzhuang1996** [reply]
I trained it in SDXL

**Reply by thiner** [reply]
try uncheck the default styles.

**Reply by zackzhuang1996** [reply]
thanks man. I did uncheck all the default styles but it didn't work too.

**Reply by FrietbakkerThjeu** [reply]
How did you train it in SDXL! even my google colab crashes when i try that - SD1.5 works well enough though.

Are there any other Loras interfering with your prompts? Are you using random seed or fixed seed?
Can you try if it works with inpaint? so for instance, if you are using a pocahontas face lora, you are not getting the pocahontas face in your generated image, try and inpaint it using "pocahontas face" prompt.

**Reply by markrmiller** [reply]
I've always found the same thing. Character Lora's that work great in automatic or comfy have minimal affect with Fooocus in comparison. Regardless of turning off styles or other Lora's or anything really.

**Reply by kalle07** [reply]
is it possible that the weights for such long prompts are calculated in a different way ?

**Reply by mashb1t** [reply]
LoRAs are handled the same way in Fooocus and ComfyUI. Can you please provide a link to mentioned character LoRA to test if it's working or not?
